pallets/payment/src/types.rs

```diff
@@ -59,3 +59,7 @@ pub trait DisputeResolver<Account> {
 	/// Get a DisputeResolver (Judge) account
 	fn get_origin() -> Account;
 }
+
+pub trait FeeHandler<Account, Asset, Amount> {
+	fn apply_fees(from: &Account, to: &Account, asset: Asset, amount: Amount) -> Amount;
```

Comment:
I'd say let's make this function return already the account that is receiving the fee instead of the pallet constant, later we can introduce the Vec thing to support multiple receivers.
I'm also hesitant about using an amount since there's more checks to make(e.g. can we withdraw that amount?) and merchants will usually just care about configurign a percentage(`-> (Account, Permill)`?).
Finally, to keep the trait simpler, at least at the begging, I think we can also ditch the `Asset` and `Amount` dependency since, at least for Virto, we only need to know the `from` and `to` to determine who gets the fee(e.g. local community or marketplace)

Reply:
Let's also create a public dummy implementation of the trait for a generic `Account` that returns the account with a 0 fee and we use it in our runtime config :)

Reply:
Created #162 to track multiple recipients improvement